### DISCUSSION TITLE
Error in starting HSRC (v23.10)

### DISCUSSION THREAD

**Discussion by NAGANOKATSUYUKI** [HumanSupportRobot/QandA#91] [Q&A]
When starting HSRC, press the power button, and when the screen with HSR letters appears, release the stop button. The LED lamp is yellow, and after the transition, the lamp remains yellow and turns red after a short time, causing the error shown in the following image.
There was a similar question (https://github.com/orgs/HumanSupportRobot/discussions/71) about this error. However, I could not find "unknown" about the communication of the amplifier.
Please, anyone can give me some ideas on how to solve this problem.
<img width="171" alt="image" src="https://github.com/HumanSupportRobot/QandA/assets/112745083/890424cc-c6a9-4f81-ac78-f056a5150759">

**Comment by NAGANOKATSUYUKI** [comment]
I downgraded to v23.04 following the clean install instructions. The same error sometimes occurs at that time, but it is fixed after a reboot. It works as usual.  I will be waiting for your information.

**Comment by yoshimi-iyoda** [comment]
I have a clean install of 23.10 on hand, and both HSRB/HSRC do not have the URG error, so I suspect the problem is hardware dependent on the robot.

Could you please check the following to see if there are any URG-related error logs?
https://docs.hsr.io/hsr_log_analysis_manual/log_files.html#journald
https://docs.hsr.io/hsr_log_analysis_manual/log_files.html#ros


In addition, the following is a useful reference for URG's failure analysis.
https://docs.hsr.io/hsrb_maintenance_manual/src/check_urg.html
